README.md

```diff
@@ -1 +1,45 @@
-# dataverse-frontend
\ No newline at end of file
+# Dataverse Frontend
+
+[![Tests](https://github.com/IQSS/dataverse-frontend/actions/workflows/test.yml/badge.svg)](https://github.com/IQSS/dataverse-frontend/actions/workflows/test.yml)
+
+## Getting Started
+
+First install node >=16 and npm >=8. Recommended versions `node v19` and `npm v9`.
+
+### `npm install`
+
+Installs dependencies.
+
+
+## Available Scripts
+
+In the project directory, you can run at any time:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner for the unit tests in the interactive watch mode.
+
+### `npm run cy:run`
+
+Launches the Cypress test runner for the end-to-end tests. \
+If you prefer to see the tests executing in cypress you can run `npm run cy:open`
```

Comment:
@JR-1991 told me that if I run `npx cypress open` I can run the tests in the Cypress GUI and see what the "hello Dataverse" webapp looks like (screenshot below). Should we document this command? It seems very useful! 😄 

<img width="1392" alt="Screen Shot 2023-02-16 at 12 03 45 PM" src="https://user-images.githubusercontent.com/21006/219437077-7ce41762-549a-4092-a2bd-99083b222bd6.png">

Reply:
Hi Philip, I don't understand what you mean. The README says 'If you prefer to see the tests executing in cypress you can run `npm run cy:open`' which is the same as the `npx cypress open`. Then is already documented in the README

Reply:
Oh! I think I simply missed `npm run cy:open` because it didn't have `###` in front of it! My mistake! 😅 

Thanks for pointing it out! I'll resolve this.